chapters/missing-data.qmd

```diff
@@ -0,0 +1,598 @@
+---
+knitr:
+  opts_chunk:
+    cache.path: "../_cache/missing-data/"
+---
+
+# Missing Data {#sec-missing-data}
+
+```{r}
+#| label: missing-data-setup
+#| include: false
+#| message: false
+#| warning: false
+
+source("../R/_common.R")
+
+# ------------------------------------------------------------------------------
+# Required packages
+
+library(tidymodels)
+library(rpart)
+library(partykit)
+
+# ------------------------------------------------------------------------------
+# Set options
+
+tidymodels_prefer()
+theme_set(theme_transparent())
+set_options()
+
+# ------------------------------------------------------------------------------
+# Load other data
+
+load("../RData/ames_original.RData")
+load("../RData/hotel_rates_all.RData")
+```
+
+```{r}
+#| label: missing-data-summaries
+#| cache: true
+#| message: false
+#| warning: false
+
+#Note:  Remove quality predictors. These are subjective and related to Sale Price
+ames_original <-
```

Comment:
I think that we should also remove "misc features" since that seems to be similar to an open text field.

Reply:
I'd like to do a mass renaming of a lot of the Ames columns so that the figures look more polished (just an FYI)

Reply:
I agree.